docker-compose.ci.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.4"
+services:
+  test:
+    build:
+      context: .
+      target: test
+    env_file: .env-test
+    depends_on:
+      - mariadb
+      - elasticsearch7
+      - redis
+
+  prod:
```

Comment:
why is this needed?

Reply:
To allow us to do `docker-compose build prod` and end up with an appropriately tagged image